pypeclub/OpenPype
# General: Anatomy moved to pipeline

## Brief description
Anatomy logic was moved from `openpype.lib` to `openpype.pipeline.anatomy`.

## Description
Anatomy class is moved to `openpype.pipeline` because is one of functionality which uses client queries, settings and lib that can cause cross import issues. It is still possible to import it from `openpype.lib` and `openpype.api` but anatomy created from those places will cause deprecated warning.

## Additional info
Didn't replace any of imports so there will be a lot of deprecated warnings all over the place. I would like to keep this PR as simple as possible and next PR or PR into this PR replacing imports. I've checked for usages of Anatomy static and class methods and found out that they're not used so it should be ok. Fixed issue with copy of `AnatomyTemplateResult`.

## Testing notes:
All functionality related to anatomy should work as did before.
1. Open host of your preference
2. Open different workfile using workfiles tool
3. Create some instances
4. Publish them
5. All should go ok
